acme/messages.py

```diff
@@ -182,15 +182,27 @@ def emails(self):
 
     @property
     def phone(self):
-        """Phone."""
-        assert len(self.phones) == 1
-        return self.phones[0]
+        """Phone.
```

Comment:
This doesn't seem right.  I don't know when it would be useful to pick any phone or any email?